velox/docs/configs.rst

```diff
@@ -129,6 +129,14 @@ Generic Configuration
      - 0
      - If it is not zero, specifies the time limit that a driver can continuously
        run on a thread before yield. If it is zero, then it no limit.
+   * - prefixsort_normalized_key_max_bytes
+     - integer
+     - 9223372036854775807
```

Comment:
This value is way too high. Let's change to a more reasonable default. How about, 128 bytes?

Reply:
@mbasmanova 
This means no-limit by default，same with duckDB(also do not limit the buffer size).

> 128 bytes
I find that it`s hard to clear & exact default value, however, our user may need to config this in future, so I add the config and set to max-int.

Reply:
@skadilover Thank you. We tend to prioritize safety and reliability over everything else. Hence, would like a safe limit.